guides/installation/src/git/git.md

```diff
@@ -0,0 +1,158 @@
+# Using Git
+
+## *Getting* Git
+
+[Check this out](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+to install Git on your computer.
+
+If you're just getting it for the first time, make sure to get your name and
+your email address.
+
+```
+$ git config --global user.name "Mr. Dudeface McGee IV"
+$ git config --global user.email "xXepic_programmer1337Xx@hotmail.com"
+```
+I'd recommend using your actual name and email and not those above.
+
+For more configuration options:
+```
+$ git config --list --show-origin # shows config file location 
+$ git config --list # shows all settings
+```
+
+## What to Know
+
+There are a ton of good resources out there on how to use Git. You shouldn't
+have a problem with finding any. At a bare minimum to make commits and things,
+you should know how to use the following:
+
+- `git add`
+- `git commit`
+- `git pull`
+- `git push`
+- `git checkout`
+- `git branch`
+- `git merge`
+- `git stash`
+ 
+### Good Resources:
+- [Atlassian Git Tutorial](https://www.atlassian.com/git/tutorials)
+- [Atlassian Cheatsheet](https://www.atlassian.com/git/tutorials/atlassian-git-cheatsheet)
+- [Interactive Git Branching](https://learngitbranching.js.org/)
+- [Stack Overflow Guide](https://stackoverflow.com/questions/315911/git-for-beginners-the-definitive-practical-guide)
+- [Pro Git Book](https://git-scm.com/book/en/v2)
+- [Official Documentation](https://git-scm.com/docs)
+
+### Branching:
+In order for us to all work together on the same project, we use Git branching
+often. This allows us to all contribute at the same time without interference,
+and for us to merge our work into the main line when it's done and has been
+verified.
+
+In most UAV Austin repositories, the main state of the code is in the *master*
+branch. For each card on Trello, generally, we will have a *feature branch* for
+it while it is in development. When it's just about finished, you'll add a
+*pull request* (or *PR*) to merge it into the *master* branch. A team lead will
+review the code real fast, make sure it's good, and then merge it in.
+
+### Merging:
+In Git, there are two types of merges:
+- fast-forward merge
+- three-way merge
+
+If you want a more in-depth explanation, click [here](https://www.atlassian.com/git/tutorials/using-branches/git-merge).
+
+**Fast-Forward Merge**
+
+A fast-forward merge occurs when the feature branch's latest common ancestor
+with the master branch is also the latest commit in the master branch. Fast-forward
+merge is enabled by default.
+
+Here's an example (switch to an appropriate directory):
+```
+$ git init
+$ touch a.txt
+$ echo "commit 1" > a.txt
+$ git add . && git commit -m "first commit"
+$ git checkout -b feature branch
```

Comment:
adding comments to each of these commands may provide clarity. Also, typo on line 77, should be "feature-branch"
```suggestion
$ git init #creates a new repository, starts you off in the master branch
$ touch a.txt #creates a new file
$ echo "commit 1" > a.txt #adds text to a file
$ git add . && git commit -m "first commit" #stages a change from the working directory to the staging area, then captures a "snapshot" of the staging area
$ git checkout -b feature-branch #creates and navigates to a new branch called "feature-branch"
```

Reply:
Fixed